src/Poe/RemoteMemoryObjects/TheGame.cs

```diff
@@ -10,7 +12,11 @@ public TheGame(Memory m)
             Address = m.ReadLong(Offsets.Base + m.AddressOfProcess, 0x8, 0xf8);//0xC40
             Game = this;
         }
-        public IngameState IngameState => ReadObject<IngameState>(Address + 0x38);
+        public IngameState IngameState => GameController.Instance.Cache.Enable ? GameController.Instance.Cache.IngameState : IngameStateReal;
+
+        public IngameState IngameStateReal => ReadObject<IngameState>(Address + 0x38);
+
         public int AreaChangeCount => M.ReadInt(M.AddressOfProcess + Offsets.AreaChangeCount);
+        public bool GameIsLoading => M.ReadInt(0x7FF6EF93F938) == 1;
```

Comment:
i don't see anything on this address??? was it valid before game update?

Reply:
@tehcheat found the AOB scan for this.
We can add this in offset.isLoadingScreen offset.
Drawback: yet another AOB scab to look after on big patches....benifit: far more reliable than game.exe + 0x473BCxxx thingy.

Reply:
I think better just delete this, because with one more AOB scan we get more time start hud for not important data.
Bug with area change happens rarely and with coroutine we can just wait after area change ~100-500 ms before doing work.
I mean problem when preload show not full data or xprate plugin get wrong xp data.

Reply:
I think maintaining a AOB scan would be better/easier than talking to people why plugin is working sometime and not working sometimes...or hey I cannot see preloads...etc etc....

I would say, let's keep it. not a big thing.